packages/react-magma-dom/src/components/Input/Input.test.js

```diff
@@ -474,4 +475,67 @@ describe('Input', () => {
       });
     });
   });
+
+  describe('Character Counter', () => {
+    const charactersAllowed = defaultI18n.characterCounter.charactersAllowed;
+    const charactersLeft = defaultI18n.characterCounter.charactersLeft;
+    const labelText = 'Character Counter';
+
+    it('Shows the label "characters allowed" equal to the maxLength if the user clears the input', () => {
+      const onChange = jest.fn();
+      const initialValue = 'dddd';
+      const { getByText, getByLabelText } = render(
+        <Input labelText={labelText} maxLength={4} onChange={onChange} />
+      );
+
+      fireEvent.change(getByLabelText(labelText), {
+        target: { value: initialValue },
+      });
+
+      expect(getByLabelText(labelText)).toHaveAttribute('value', initialValue);
+      expect(getByText('0 ' + charactersLeft)).toBeInTheDocument();
+
+      fireEvent.change(getByLabelText(labelText), {
+        target: { value: '' },
+      });
+
+      expect(getByLabelText(labelText)).toHaveAttribute('value', '');
+      expect(getByText('4 ' + charactersAllowed)).toBeInTheDocument();
+    });
+
+    describe('styling', () => {
```

Comment:
I think styling tests can move to CharacterCounter as they are not Input specific

Reply:
I did the same thing in 'Textarea.test.js', should that also be removed?

Reply:
Yeah! This is the kind of test that we only need once if we put it in CharacterCounter.